### ISSUE TITLE
Cannot load loader script

### ISSUE THREAD

**Issue by andresfelipe9619** [boundlessgeo/sdk-apps#29]
i found myself in trouble after 3 hours of no succes with the get started i follow all the steps you have there
 `npm run build
  npm start
  npm run package`
...and everything looks fine but when i open the browser in localhost:3000, there's nothig to see and i get in the console this log
Ha fallado la carga del <script> con origen "http://localhost:3000/loader.jsC:/Users/HP/Desktop/Proyecto%20Web%20Mapping%20Cantera/sdk-examples/sdk-apps/quickview/node_modules/openlayers/src/ol/view.js".
localhost:3000:29
Ha fallado la carga del <script> con origen "http://localhost:3000/loader.jsC:/Users/HP/Desktop/Proyecto%20Web%20Mapping%20Cantera/sdk-examples/sdk-apps/quickview/node_modules/openlayers/src/ol/kinetic.js".
localhost:3000:29
Ha fallado la carga del <script> con origen "http://localhost:3000/loader.jsC:/Users/HP/Desktop/Proyecto%20Web%20Mapping%20Cantera/sdk-examples/sdk-apps/quickview/node_modules/openlayers/src/ol/interaction/property.js".
localhost:3000:29
Ha fallado la carga del <script> con origen "http://localhost:3000/loader.jsC:/Users/HP/Desktop/Proyecto%20Web%20Mapping%20Cantera/sdk-examples/sdk-apps/quickview/node_modules/openlayers/src/ol/interaction/interaction.js".
localhost:3000:29
Ha fallado la carga del <script> con origen "http://localhost:3000/loader.jsC:/Users/HP/Desktop/Proyecto%20Web%20Mapping%20Cantera/sdk-examples/sdk-apps/quickview/node_modules/openlayers/src/ol/interaction/doubleclickzoom.js".
localhost:3000:29
Ha fallado la carga del <script> con origen "http://localhost:3000/loader.jsC:/Users/HP/Desktop/Proyecto%20Web%20Mapping%20Cantera/sdk-examples/sdk-apps/quickview/node_modules/openlayers/src/ol/events/condition.js".
localhost:3000:29
Ha fallado la carga del <script> con origen "http://localhost:3000/loader.jsC:/Users/HP/Desktop/Proyecto%20Web%20Mapping%20Cantera/sdk-examples/sdk-apps/quickview/node_modules/openlayers/src/ol/interaction/pointer.js".
localhost:3000:29
Ha fallado la carga del <script> con origen "http://localhost:3000/loader.jsC:/Users/HP/Desktop/Proyecto%20Web%20Mapping%20Cantera/sdk-examples/sdk-apps/quickview/node_modules/openlayers/src/ol/interaction/dragpan.js".
localhost:3000:29
Ha fallado la carga del <script> con origen "http://localhost:3000/loader.jsC:/Users/HP/Desktop/Proyecto%20Web%20Mapping%20Cantera/sdk-examples/sdk-apps/quickview/node_modules/openlayers/src/ol/interaction/dragrotate.js".
localhost:3000:29
Ha fallado la carga del <script> con origen "http://localhost:3000/loader.jsC:/Users/HP/Desktop/Proyecto%20Web%20Mapping%20Cantera/sdk-examples/sdk-apps/quickview/node_modules/openlayers/src/ol/render/box.js".

Dont know if im doing something wrong, or missing some step but please helpe me to get started! than you

**Comment by bartvde** [comment]
are you using firefox on windows? I think the openlayers loader on which the debug server is based has issues.

Btw sdk 1.x on which this repo is based is kind of deprecated and we encourage you to use sdk 2.x

**Comment by andresfelipe9619** [comment]
yes, im using firefox in windows. I'll try it with linux and let you know what happened. thanks for your time

**Comment by bartvde** [comment]
yes either that or try chrome on windows
